cli/src/cli.rs

```diff
@@ -48,10 +48,6 @@ pub enum Subcommand {
 		about = "Decode given block or extrinsic using current native runtime."
 	)]
 	Inspect(node_inspect::cli::InspectCmd),
-
-	/// The custom benchmark subcommmand benchmarking runtime pallets.
```

Comment:
https://github.com/cennznet/cennznet/issues/178

Reply:
Have added comments with tracking issue to these parts. Will push if after build passes